packages/tools/src/utilities/rectangleROITool/isAxisAlignedRectangle.ts

```diff
@@ -0,0 +1,62 @@
+import { vec3 } from 'gl-matrix';
+import { utilities as csUtils } from '@cornerstonejs/core';
+
+/**
+ * Determines whether a given rectangle in a 3D space (defined by its corner
+ * points in IJK coordinates) is aligned with the IJK axes.
+ * @param rectangleCornersIJK - The corner points of the rectangle in IJK coordinates
+ * @returns True if the rectangle is aligned with the IJK axes, false otherwise
+ */
+function isAxisAlignedRectangle(rectangleCornersIJK) {
+  const iAxis = [1, 0, 0];
```

Comment:
Use vec3 vectors and leave them defined outside the function as they are constants.